app/models/release_artifact.rb

```diff
@@ -453,9 +453,11 @@ class ReleaseArtifact < ApplicationRecord
   scope :yanked,    -> { joins(:release).where(releases: { status: 'YANKED' }) }
   scope :unyanked,  -> { joins(:release).where.not(releases: { status: 'YANKED' }) }
 
-  scope :gems, -> { for_filetype(:gem) }
+  scope :gems,             -> { for_engine(:rubygems).for_filetype(:gem) }
+  scope :npm_package_tgz,  -> { for_engine(:npm).for_filetype(:tgz) }
```

Comment:
Should we just call this `tarballs` here and drop the engines? Could include `tar`, `tgz`, `tar.gz`, `tar.zz`, `tar.bz2`, etc.